pkg/types/baremetal/validation/platform.go

```diff
@@ -326,18 +325,6 @@ func validateHostsCount(hosts []*baremetal.Host, installConfig *types.InstallCon
 	return nil
 }
 
-// ensure that the NetworkConfig field contains a valid Yaml string
-func validateNetworkConfig(hosts []*baremetal.Host, fldPath *field.Path) (errors field.ErrorList) {
```

Comment:
I think we still want this validation that the network config can be unmarshaled into a `map[string]interface{}`. The apiextensions JSON can accept any valid JSON value.
```
// JSON represents any valid JSON value.
// These types are supported: bool, int64, float64, string, []interface{}, map[string]interface{} and nil.
```

Reply:
Yep, good idea